test/powershell/Modules/Microsoft.PowerShell.Security/CertificateProvider.Tests.ps1

```diff
@@ -292,4 +292,80 @@ Describe "Certificate Provider tests" -Tags "Feature" {
             $certs.Thumbprint | Should -BeExactly $thumbprint
         }
     }
+
+    Context "SAN DNS Name Tests" {
+        BeforeAll {
+            $configFilePath = Join-Path -Path $TestDrive -ChildPath 'openssl.cnf'
+            $keyFilePath = Join-Path -Path $TestDrive -ChildPath 'privateKey.key'
+            $certFilePath = Join-Path -Path $TestDrive -ChildPath 'certificate.crt'
+            $pfxFilePath = Join-Path -Path $TestDrive -ChildPath 'certificate.pfx'
+            $password = "test"
```

Comment:
Please generate random password (we have compliance requirements).

Reply:
Ah good catch. I forgot to do this once I got the tests working 😂

Reply:
@iSazonov Fixed in 18a73cd1e